### PR TITLE
chore: update asset bundle URLs

### PR DESCRIPTION
**Pull request by pravusjif**
Updated urls to fetch asset bundles

------------------------------------------------------

Now we shouldn't have the Asset Bundle conversion/fetch errors

### CODE REVIEW

**Review by eordano — APPROVED** [review]

File: kernel/packages/config/index.ts [hunk 1]
```diff
@@ -228,7 +228,7 @@ export function getServerConfigurations() {
     auth: `https://auth.decentraland.${TLDDefault}/api/v1`,
     landApi: `https://api.decentraland.${TLDDefault}/v1`,
     content: `https://content.decentraland.${TLDDefault === 'today' ? 'org' : TLDDefault}`,
-    contentAsBundle: `https://content-as-bundle.decentraland.zone`,
+    contentAsBundle: `https://content-assets-as-bundle.decentraland.org`,
```
Comment (BrianAmadori) [inline kernel/packages/config/index.ts:231]: `content-assets-as-bundle` is supposed to be the name of the bucket, not the content server layer. Are you sure this is correct?

Comment (BrianAmadori) [reply kernel/packages/config/index.ts:231]: @marcosnc @adrabenche

Comment (pravusjif) [reply kernel/packages/config/index.ts:231]: Brian apparently that's the new URL, you can check it with this example:
https://content-assets-as-bundle.decentraland.org/QmNRKtcJJmQgtNom3Ug9ZcycCtHggs3HQPGxHPp8D5m1uF

**Comment by pravusjif** [comment]
I'm also asking the comms/content guys because I see in the build we have a CORS problem fetching asset bundles from zone
```
Access to XMLHttpRequest at 'https://content-assets-as-bundle.decentraland.org/QmRVH457TzrYjqZSjnmszkbA9xRU4myNJQTbwpQ3yUrBaH' from origin 'https://explorer.decentraland.zone' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```

**Comment by pravusjif** [comment]
We just resolved the CORS problem using the following config:

`<CORSConfiguration>
<CORSRule>
    <AllowedOrigin>https://explorer.decentraland.zone</AllowedOrigin>
    <AllowedOrigin>https://explorer.decentraland.org</AllowedOrigin>
    <AllowedMethod>GET</AllowedMethod>
    <AllowedMethod>HEAD</AllowedMethod>
    <MaxAgeSeconds>3000</MaxAgeSeconds>
    <AllowedHeader>*</AllowedHeader>
    <ExposeHeader>ETag</ExposeHeader>
</CORSRule>
</CORSConfiguration>`

**Review by BrianAmadori — APPROVED** [review]

**Pull request merged**